recipes/jupyterlab-spellchecker/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "jupyterlab-spellchecker" %}
+{% set version = "0.5.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyterlab-spellchecker-{{ version }}.tar.gz
+  sha256: c2feb33273810cfd54271a29eaed1ffdb88d3ef0b0a5d33495b5954f164ee269
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - jupyter-packaging
+    - pip
+    - python >=3.6
+  run:
+    - jupyterlab ~=3.0
+    - python >=3.6
+
+test:
+  imports:
+    - jupyterlab_spellchecker
+  commands:
+    - pip check
```

Comment:
it's nice to do a sanity check as to whether the labextension is in place and found by the app, e.g. in `$PREFIX/share`, e.g.

```yaml
- jupyter labextension list 1>labextensions 2>&1
- cat labextensions | grep "jupyterlab-lsp.*OK.*jupyterlab_lsp"  # [unix]
```

(modified for `grep <@npm/package>.*OK<python_module>`)

alternately, this can be done in python, but given this kinda fits in one file...

I wouldn't bother with the `browsercheck`, as it just kinda dies (for many reasons, e.g. docker) and doesn't tell much when it does: `there were errors`.

Reply:
Added, thanks!